homeassistant/components/drop_connect/entity.py

```diff
@@ -0,0 +1,29 @@
+"""Base entity class for DROP entities."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import DROPDeviceDataUpdateCoordinator
+
+
+class DROPEntity(CoordinatorEntity[DROPDeviceDataUpdateCoordinator]):
+    """Representation of a DROP device entity."""
+
+    _attr_force_update = False
+    _attr_has_entity_name = True
+    _attr_should_poll = False
+
+    def __init__(
+        self, entity_type: str, coordinator: DROPDeviceDataUpdateCoordinator
+    ) -> None:
+        """Init DROP entity."""
+        super().__init__(coordinator)
+        self._attr_unique_id = f"{coordinator.unique_id}_{entity_type}"
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, self.coordinator.unique_id)},
+            manufacturer=self.coordinator.manufacturer,
+            model=self.coordinator.model,
+            name=self.coordinator.device_name,
+        )
```

Comment:
```suggestion
        self._attr_device_info = DeviceInfo(
            identifiers={(DOMAIN, coordinator.unique_id)},
            manufacturer=coordinator.manufacturer,
            model=coordinator.model,
            name=coordinator.device_name,
        )
```

Reply:
Fixed in a7d5542787d34061329608f8bca4528568414bb3.